content/en/docs/reference/api/cert-link-requests/_index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Certificate linking requests API"
+linkTitle: "Certificate linking requests API"
+---
+
+
+IAM Login Service provides a RESTful API to create and manage certificate linking requests.
+
+## POST /iam/group_requests
+
+Create a new certificate linking request.
+The body of the request must contain a JSON with the following parameters:
+
+| Name | Values | Optional or Required | Description |
+|:------------:|-------------|-------------|-------------|
+| **label**<br/>`string`| A non empty string. | Required | Certificate label. |
```

Comment:
Some sentences have a full stop at the end, other don't. I'd remove it from all of them.

Reply:
Done